src/System.Management.Automation/engine/CommandBase.cs

```diff
@@ -8,7 +8,7 @@
 using System.Management.Automation.Internal.Host;
 using System.Management.Automation.Language;
 using System.Management.Automation.Runspaces;
-
+using System.Threading;
 using Dbg = System.Management.Automation.Diagnostics;
```

Comment:
nipick: can you please remove the `using Dbg =...` line? I think it's not used.

Reply:
Looks like it is on line 1304 and 1315.

https://github.com/PowerShell/PowerShell/blob/1b03005d1ac669e5089cc96e707042b0c9bb8883/src/System.Management.Automation/engine/pipeline.cs#L1302-L1328

Do you want me to keep it, remove the alias in favour of a normal using or using the full type line on those lines?

Reply:
Can do.